puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -73,6 +73,11 @@
   $n1kv_vsm_ip                   = undef,
   $n1kv_vsm_password             = undef,
   $n1kv_vsm_username             = undef,
+  $plumgrid_director_vip         = undef,
```

Comment:
The patch does not cause errors when using ml2, however, if you want these parameters to be overridden in the control hostgroup, you need to surface these same parameters in quickstack::pacemaker::neutron.  Also, is this _vip param an actual pacemaker vip?  If so, you will need to create that vip in the  quickstack::pacemaker::neutron class as well. If not, what is it?